### ISSUE TITLE
recalculate hourly rate

### ISSUE THREAD

**Issue by yegor256** [zerocracy/farm#1459]
Let's recalculate the hourly rate of a programmer automatically, every day. I'm talking about the rate we keep in `peopoe.xml`, not the project rate. Let's use our internal metrics and re-set the rate according to them. This article should help: https://www.yegor256.com/2014/04/20/how-hourly-rate-is-calculated.html When the rate is changed, let's inform the developer via `Notify user`. Let's make the numbers "round" to an integer.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@yegor256 do you have some coefficients for these metrics? How exactly each metric should affect the rate?

**Comment by 0crat** [comment]
Job [#1459](https://github.com/zerocracy/farm/issues/1459) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by llorllale** [comment]
@yegor256 @g4s8 

As PO or ARC, why wouldn't I want to set the DEV rates as I see fit for the project?

**Comment by yegor256** [comment]
@llorllale yes, you can. Here we are talking about the profile rate, which you set for yourself via Telegram. The rate in the project is set by the ARC/PO, not the user and not the bot.
